tempwork.js

```diff
@@ -74,7 +74,7 @@ function initializeWorker(name) {
 }
 function restartWorker(name) {
   var worker = workers[name];
-  var env = {};
+  var env = process.env;
```

Comment:
Can we use `Object.assign(process.env)` to keep from ever mutating the host process environment?